bin/strata-client/src/el_sync.rs

```diff
@@ -70,6 +70,55 @@ pub fn sync_chainstate_to_el(
     Ok(())
 }
 
+/// Sync missing or invalid L2 block status using chainstate.
+///
+/// TODO: retry on network errors
```

Comment:
These two lines should be comments not docstrings

```suggestion
//
// TODO: retry on network errors
```